.gitignore

```diff
@@ -2,3 +2,5 @@ node_modules
 dist
 src/version.json
 tmp
+-p
```

Comment:
This `-p` addition in `.gitignore` seems strange, is that a hiccup for copying and pasting from `mkdir -p dist`?